sklearn/linear_model/base.py

```diff
@@ -523,7 +523,7 @@ def _pre_fit(X, y, Xy, precompute, normalize, fit_intercept, copy):
         precompute = False
         X, y, X_offset, y_offset, X_scale = _preprocess_data(
             X, y, fit_intercept=fit_intercept, normalize=normalize,
-            return_mean=True)
+            copy=copy, return_mean=True)
```

Comment:
Ok, got it. Let me change to copy=False.
Read through this now
    copy : boolean (default=False)
        Whether a forced copy will be triggered. If copy=False, a copy might
        be triggered by a conversion.